forge/files/src/index.js

```diff
@@ -39,7 +39,9 @@ app.on('ready', async () => {
       console.log('Failed to install Devtron: ', err);
     }
     try {
-      await installExtension(EMBER_INSPECTOR);
+      await installExtension(EMBER_INSPECTOR, {
+        loadExtensionOptions: { allowFileAccess: true },
```

Comment:
Should this be `loadExtensionOptions: { allowFileAccess: true }` or just `{ allowFileAccess: true }`?

Reply:
It's needed. `loadExtensionOptions` is what electron-devtools-installer passes to the options argument of electron's `loadExtension` method, which is where the allowFileAccess option is now available.